pkg/controllers/securitypolicy_controller.go

```diff
@@ -78,6 +79,15 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return resultNormal, client.IgnoreNotFound(err)
 	}
 
+	// Since securitypolicy service can only be activated from nsx 3.2.0 onwards,
+	// So need to check nsx version before starting securitypolicy Reconcile
+	if !r.Service.NSXClient.CheckNsxVersion() {
+		err := errors.New("check Nsx Version failed, would retry after 5 minutes")
```

Comment:
`would retry after 5 minutes`: I don't think we need to say retry 5 minutes

Reply:
how about turning it into a function to support various interval time?